tests/src/test/scala/ch/epfl/bluebrain/nexus/tests/kg/SearchConfigSpec.scala

```diff
@@ -87,6 +91,77 @@ class SearchConfigSpec extends BaseSpec {
       }
     }
 
+    "have the correct name property from schema:name" in {
+      val query = queryField(neuronMorphologyId, "name")
+      assertOneSource(query) { json =>
+        json shouldEqual json"""{ "name": "sm080522a1-5_idA" }"""
+      }
+    }
+
+    "have the correct name property from rdfs:label" in {
+      val query = queryField(neuronDensityId, "name")
+      assertOneSource(query) { json =>
+        json shouldEqual json"""{ "name": "Neuron density: CA1" }"""
+      }
+    }
+
+    "have the correct name property from skos:prefLabel" in {
+      val query = queryField(traceId, "name")
+      assertOneSource(query) { json =>
+        json shouldEqual json"""{ "name": "S1J_L6_IPC_cADpyr_2" }"""
+      }
+    }
+
+    "have the correct description property" in {
+      val query = queryField(neuronMorphologyId, "description")
+      assertOneSource(query) { json =>
+        json shouldEqual json"""{ "description": "This is a resource description." }"""
+      }
+    }
+
+    "have the correct createdAt property" in {
+      val query = queryField(neuronMorphologyId, "createdAt")
+      assertOneSource(query) { json =>
+        assert(isInstant(json, "createdAt"))
```

Comment:
Just checking that the value can be parsed as an `Instant` for `createdAt` and `updatedAt`